File: Fizzbuzz.rb
def fizzbuzz(min_value, max_value)
	while min_value <= max_value
		if min_value % 5 == 0 and min_value % 3 == 0
			puts "[#{min_value}] - Fizzbuzz!"
		elsif min_value % 3 == 0
			puts "[#{min_value}] - Fizz!"
		elsif min_value % 5 == 0
			puts "[#{min_value}] - Buzz!"
		else
			puts "[#{min_value}]"
		end
		min_value += 1
	end
end
puts "Minimal Value: "; min_value = gets.chop.to_i
puts "Maximal Value: "; max_value = gets.chop.to_i
fizzbuzz(min_value, max_value)

File: Fizzbuzz.py
def fizzbuzz(int_start, int_end):
	print("fizzbuzz:")
	for i in range(int_start, int_end+1):
		if i % 3 == 0 and i % 5 == 0:
			print(f"[{i}] - Fizzbuzz!")
		elif i % 3 == 0:
			print(f"[{i}] - Fizz!")
		elif i % 5 == 0:
			print(f"[{i}] - Buzz!")
		else:
			print(f"[{i}]")

minIN = int(input("Minimal Value: "))
maxIN = int(input("Maximal Value: "))
fizzbuzz(minIN,maxIN)